spearhead/assets/sass/_author-bio.scss

```diff
@@ -0,0 +1,22 @@
+.author-link {
+	font-size: var(--global--font-size-xs);
+	position: relative;
+
+	&:after {
```

Comment:
Maybe we should make this a mixin since we are starting to reuse it?

Reply:
I'm not sure there are quite a few differences. The position is bottom right not top left for example...